content/freeze.md

```diff
@@ -34,9 +34,10 @@ An account cannot enable the Individual Freeze setting if it has previously enab
 Global Freeze
 -------------
 
-The **Global Freeze** feature is a setting on an account. When an issuing account enables the Global Freeze feature, the following rules apply:
+The **Global Freeze** feature is a setting on an account. [//]: # (Is this not the case for the other Freeze settings?) When an issuing account enables the Global Freeze feature, the following rules apply:
```

Comment:
Global Freeze and No Freeze are settings on an account.
Individual Freeze is a setting on a trust line.

So, in short, yes.